.changeset/four-beers-smell.md

```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/sidebar-settings": patch
```

Comment:
cli missing, but would splitted them to another PR so you don't need that many approvals